_includes/seo.html

```diff
@@ -113,7 +115,7 @@
   <script type="application/ld+json">
     {
       "@context" : "http://schema.org",
-      "@type" : "{% if site.social.type %}{{ site.social.type }}{% else %}person{% endif %}",
+      "@type" : "{% if site.social.type %}{{ site.social.type }}{% else %}Person{% endif %}",
```

Comment:
Actually [per the schema spec](http://schema.org/Person) I believe this is suppose to be capitalized.

[`jekyll-seo-tag`](https://github.com/jekyll/jekyll-seo-tag) suggests the capitalized version as well.

> `type` - The type of things that the page represents. This must be a [Schema.org type](http://schema.org/docs/schemas.html), and will probably usually be something like [`BlogPosting`](http://schema.org/BlogPosting), [`NewsArticle`](http://schema.org/NewsArticle), [`Person`](http://schema.org/Person), [`Organization`](http://schema.org/Organization), etc.